README.md

```diff
@@ -32,8 +34,8 @@ support privileged extension, root installation, or local repositories.
 
 ## Building and Installing
 
-Android SDK path should be specified either using `ANDROID_HOME` environment variable or using `sdk.dir` property in
-`local.properties`.
+To join the development, specify your Android SDK path either using the `ANDROID_HOME` environment variable,
+or by filling out the `sdk.dir` property in `local.properties`.
```

Comment:
> "To join the development" is not correct. What should they do if they don't want to join the development, but just want to build it?

> The purpose of this block is to describe how to build and install the package, that's all. It's important because GPL actually requires such explanation, so it's placed into readme.

> As I said last time reviewing this block, I prefer my version, because this part is purely technical, and shouldn't imply anything.

Reply:
And conversely, should people not wanting to join the development remove any property in "sdk.dir"? ;)
The idea was to not scare off those that don't know. I'll think of a combination.

Reply:
But section is named "building and installing", so this section is definitely for, let's say, __hackers__. This is why I want to keep it this way.

> I actually prefer my readme structure. It's more serious, or something like that.

You're suggesting to change it to make it more "user-friendly" and interesting, and I can agree that it's the case for "Description" block (as well as its "Features" and "Screenshots" subblocks) (And they will also be displayed on F-Droid application page). But it's not the case for the rest text.

Reply:
Entirely correct, I have corrected it.

Reply:
Also, are you sure comma is necessary before "or by filling out"?

Reply:
@kitsunyan It is a natural pause. Without the comma it is more of a nor-gate logic where both together are good.

Reply:
> It is a natural pause

Hm, ok.